extensions/reviewed/CrazyGamesAdApi.json

```diff
@@ -561,6 +627,291 @@
+                "    if (error) {",
+                "        logger.error(\"Error on request banner.\", error);",
+                "    } else {",
+                "        // result is always undefined when requesting banners",
+                "        logger.log(\"End request banner\", result === undefined ? \"Requesting banners\" : result);",
+                "        eventsFunctionContext.task.resolve();",
+                "    }",
+                "});"
+              ],
+              "parameterObjects": "",
+              "useStrict": true,
+              "eventsSheetExpanded": true
+            }
+          ]
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Banner's name",
+          "name": "BannerName",
+          "type": "string"
+        },
+        {
+          "description": "Ad size",
+          "name": "Size",
+          "supplementaryInformation": "[\"728x90\",\"300x250\",\"320x50\",\"468x60\",\"320x100\"]",
+          "type": "stringWithSelector"
+        },
+        {
+          "description": "Position X",
+          "name": "PositionX",
+          "type": "expression"
+        },
+        {
+          "description": "Position Y",
+          "name": "PositionY",
+          "type": "expression"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide a banner",
+      "fullName": "Hide a banner",
+      "functionType": "Action",
+      "group": "Banner",
+      "name": "HideBanner",
+      "sentence": "Hide the banner: _PARAM1_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "const bannerName = eventsFunctionContext.getArgument(\"BannerName\");\r",
+            "window.CrazyGames.SDK.banner.clearBanner(bannerName);\r",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [
+        {
+          "description": "Banner's name",
+          "name": "BannerName",
+          "type": "string"
+        }
+      ],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide all banners",
+      "fullName": "Hide all banners",
+      "functionType": "Action",
+      "group": "Banner",
+      "name": "HideAllBanners",
+      "sentence": "Hide all the banners",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "window.CrazyGames.SDK.banner.clearAllBanners();\r",
+            ""
+          ],
+          "parameterObjects": "",
+          "useStrict": true,
+          "eventsSheetExpanded": false
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Hide the invite button.",
+      "fullName": "Hide invite button",
+      "functionType": "Action",
+      "group": "Invite",
+      "name": "HideInviteButton",
```

Comment:
Is there an action to display the invite button?

Reply:
Yes this action 
![image](https://github.com/GDevelopApp/GDevelop-extensions/assets/1670670/0cef095c-4eda-4a5a-a1be-702bd5348e4c)